### DISCUSSION TITLE
Trouble setting up dbt_projects.yml

### DISCUSSION THREAD

**Discussion by zappabad** [rittmananalytics/ra_data_warehouse#22] [Q&A]
Hi! I've stumbled onto this project a few days ago, and have been trying to setup a simple test for Facebook Ads reporting, with the data coming from Stitch, through BigQuery, but I can't seem to find where I'm doing something wrong.

I've been trying to look upon other forks to see how people setup theirs, but it seems that the new dbt_project.yml file is different from everyone else.

[This is my dbt_project.yml file](https://github.com/zappabad/ra_data_warehouse_for_bbm/blob/testing_dbt_project_yml/dbt_project.yml). The error I get when trying to compile is

`Compilation Error in model stg_facebook_ads_ad_groups (models/sources/stg_facebook_ads/bigquery/stitch/stg_facebook_ads_ad_groups.sql)
  Model 'model.ra_data_warehouse.stg_facebook_ads_ad_groups' (models/sources/stg_facebook_ads/bigquery/stitch/stg_facebook_ads_ad_groups.sql) depends on a source named 'stitch_facebook_ads.ad_sets' which was not found`

Where am I going wrong here? I don't know where to find the named source in order to track the issue properly, so if I could get any help in doing that, it would be much appreciated! The project you guys have going on seems amazing, and I really want to test it out!

**Comment by markrittman** [comment]
Hi! @markrittman here and I put this part of the framework together. I'll take a look and set-up a test example myself to see where the issue is, there's probably an assumption somewhere in the code that fails when just FB Ads is used as the source, I'll get back to you shortly with some more info. 

Mark

**Comment by markrittman** [comment]
Hi @zappabad there was an error with the source name in the dbt_project.yml file, a typo from the dev process. I also fixed the column definitions for the various tables in the stg_facebook_ads_stitch folder; background is that we primarily work with Segment as the pipeline source for Facebook and Google Ads and the stitch adapter hadn't been updated recently, hence being a bit out of date. Should work ok now though.
